crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -1193,6 +1268,7 @@ pub mod wasm_test {
     }
 
     #[tokio::test(flavor = "multi_thread")]
+    #[ignore]
```

Comment:
is it because it's flaky or some other reason? we want to find the flaky tests easily.

```suggestion
    #[ignore = "flaky"]
```

Reply:
@maackle because metering is disabled currently as it stopped working upstream